nucliadb/nucliadb/ingest/settings.py

```diff
@@ -88,7 +88,7 @@ class Settings(BaseSettings):
     sidecar_port_map: Dict[int, int] = {}
 
     # Node limits
-    max_node_fields: int = 200000
+    max_shard_fields: int = 200000  # max number of fields to target per shard
     max_node_shards: int = 600
```

Comment:
Maybe that should be named `max_node_replicas` then?

From ingest point of view (and in my mind) a shard is a logic shard. A replica shard is the container unit from the node's point of view

Reply:
it probably shouldn't be "max" either then right? It's the number that we always use. So just "shard_replicas"?

Reply:
Just to clarify: `max_node_shards` here does not mean the number of replicas for a given shard.
It means the maximum number of shard replicas that a node can have

Reply:
yeah, I'm dumb. I see now. I think it makes sense